tools/ghpages_documentation_deploy.sh

```diff
@@ -42,7 +42,7 @@ cd $TARGET_REPOSITORY_NAME
 
 echo "Replacing $TARGET_DOC_DIR with the new contents ..."
 git rm -rf $TARGET_DOC_DIR
-git rm -rf $TARGET_DOC_DIR_JA
+# git rm -rf $TARGET_DOC_DIR_JA
```

Comment:
Want to keep this line commented for any reason? if not, I'd just remove. If you want to keep it writing a comment explaining why this line is being disabled could help.